.claude/agents/changelog-generate.md

```
@@ -0,0 +1,52 @@
+tools: Glob, Grep, Read, Edit, Write, WebFetch
+model: sonnet
+color: yellow
+skills: changelog-generation
+---
+
+# Agent: Changelog Generate
+
+**Purpose**: Generate a properly formatted changelog entry for the React Native SDK release.
+
+## Input
+| Variable | Required | Description | Example |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)